.github/workflows/convention-develocity-gradle-plugin-verification.yml

```diff
@@ -0,0 +1,64 @@
+name: Verify Convention Develocity Gradle Plugin
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Convention Develocity Gradle Plugin
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up JDK 8
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Build with Gradle
+        working-directory: convention-develocity-gradle-plugin/plugins/gradle-5-or-newer
```

Comment:
Should we add a similar verification that at least builds the gradle <4 version? Even if we can't verify the functionality works, knowing it compiles is at least something

Reply:
Yes. Good point.

Reply:
Done in 4cdccf04e6f19c6998bf2e27b0e2b5b820855021